tools/embedded-zookeeper/pom.xml

```diff
@@ -31,8 +31,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
Upgrade the version here is not a good idea now

Reply:
Do not change it in this PR

Reply:
Then I will roll back the modifications in this place, and what about the rest?

Reply:
done